lib/boku/client.py

```diff
@@ -282,6 +290,20 @@ def check_transaction(self, transaction_id):
             'paid': Decimal(tree.find('paid').text),
         }
 
+    def check_sig(self, data):
```

Comment:
I think ``get_sig()`` would be a better name for this function. Also, I think the docstring is wrong about it checking the signature. It looks like ``utils.check_sig()`` does that now, right?